Source/WebCore/page/text-extraction/TextExtraction.cpp

```diff
@@ -1511,13 +1538,12 @@ InteractionDescription interactionDescription(const Interaction& interaction)
         }
     }
 
-    if (auto location = interaction.locationInRootView) {
-        auto roundedLocation = roundedIntPoint(*location);
-        description.append(" at coordinates ("_s, roundedLocation.x(), ", "_s, roundedLocation.y(), ')');
-    }
+    auto appendElementString = [&](auto&&... args) {
+        auto elementString = textDescription(std::forward<decltype(args)>(args)...);
+        if (elementString.isEmpty())
+            return;
```

Comment:
Nit: the style guide prefers an explicit template argument list when the parameter type is required in the function body.

```suggestion
    auto appendElementString = [&]<typename... T>(T&&... args) {
        auto elementString = textDescription(std::forward<T>(args)...);
        if (elementString.isEmpty())
            return;
```

Reply:
TIL — thanks for the reviews!